### PR TITLE
Added REST API

### PR DESCRIPTION
**Pull request by sheikhuzairhussain**

### CODE REVIEW

File: api/index.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            Email: req.body.email,
+            Phone: req.body.phone,
+            City: req.body.city,
+            Company: req.body.company,
+            Experience: req.body.experience,
+        },
+    ]);
+
+    res.json({
+        message: "Application submitted successfully.",
```
Comment (KaneDenzil) [inline api/index.js:33]: It'll be great if you send an error code so when the FrontEnd compares the response we can use the error code as comparison

File: api/index.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    await worksheet.addRows([
+        {
+            Name: req.body.name,
+            Email: req.body.email,
+            Phone: req.body.phone,
+            City: req.body.city,
+            Company: req.body.company,
+            Experience: req.body.experience,
+        },
+    ]);
```
Comment (KaneDenzil) [inline api/index.js:30]: How do you handle if any of the field is empty or if the type of the field is invalid

**Review by KaneDenzil — CHANGES_REQUESTED** [review]
Please let me know about the questions

File: api/index.js (hunk 2 above)
Comment (sheikhuzairhussain) [reply api/index.js:30]: This is just a barebones implementation of the idea we discussed. I will be adding proper validation and error handling over the next few commits.

Comment (KaneDenzil) [reply api/index.js:30]: Sounds good

**Pull request merged**